android/src/main/java/com/AirMaps/AirMapView.java

```diff
@@ -100,6 +110,20 @@ public boolean onScroll(MotionEvent e1, MotionEvent e2, float distanceX, float d
         });
 
         eventDispatcher = context.getNativeModule(UIManagerModule.class).getEventDispatcher();
+
+        this.mapLoadingTextView = new TextView(context);
+        this.mapLoadingTextView.setText("Loading map...");
+        this.mapLoadingTextView.setBackgroundColor(Color.parseColor("#eeeeee"));
```

Comment:
also ideally this color isn't hard coded... maybe we expose it as a prop with this as a default?